### PR TITLE
docs: new hosted_phoenix tutorials

### PR DESCRIPTION
**Pull request by exiao**
I wanted to add two new .ipynb stored on repo instead of being stored in my personal GitHub account.

### CODE REVIEW

**Comment by mikeldking** [comment]
@exiao thanks for the PR. Could you address the CI failures? The procedure for fixing the notebook formats should be in the DEVELOPMENT guide. Let us know if you get stuck

**Comment by exiao** [comment]
I have read the CLA Document and I hereby sign the CLA.

**Comment by exiao** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by exiao** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by mikeldking** [comment]
recheck

**Review by Jgilhuly — APPROVED** [review]
Looks great!

File: tutorials/hosted_phoenix/hosted_phoenix_llamaindex_tutorial.ipynb [hunk 1]
```diff
@@ -0,0 +1,254 @@
+{
```
Comment (mikeldking) [inline tutorials/hosted_phoenix/hosted_phoenix_llamaindex_tutorial.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #6.    phoenix_df = px_client.get_spans_dataframe()
</pre><p>the name here is a bit confusing. Why not spans_df?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/4018/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

**Review by mikeldking — CHANGES_REQUESTED** [review]
The examples should use project name but I don't see the corresponding code. Also do you think it's worth having a more comprehensive tutorial that includes evals? Otherwise these tutorials could be consolidated as they just show you how to setup endpoints and headers. There's no real need for both

File: tutorials/hosted_phoenix/hosted_phoenix_openai_tutorial.ipynb [hunk 2]
```diff
@@ -0,0 +1,516 @@
+{
```
Comment (mikeldking) [inline tutorials/hosted_phoenix/hosted_phoenix_openai_tutorial.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #19.    px_client.get_spans_dataframe(project_name="rechat", root_spans_only=True)
</pre><p>I don't see project name in any of the code above. You sure this works? also why the root_spans_only flag here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/4018/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (mikeldking) [inline tutorials/hosted_phoenix/hosted_phoenix_openai_tutorial.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #16.        dataset_name="test-dataset-2",
</pre><p>Can you make this dataset more realistic?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/4018/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

**Comment by mikeldking** [comment]
These need work - but they are better than not having them. Will work with @Jgilhuly to freshen these up

**Pull request merged**